stdlib/tarfile.pyi

```diff
@@ -264,6 +264,7 @@ class TarFile:
     def extract(
         self, member: Union[str, TarInfo], path: StrOrBytesPath = ..., set_attrs: bool = ..., *, numeric_owner: bool = ...
     ) -> None: ...
+    def _extract_member(self, tarinfo: TarInfo, targetpath: str, set_attrs: bool = ..., numeric_owner: bool = ...) -> None: ...
```

Comment:
```suggestion
    def _extract_member(self, tarinfo: TarInfo, targetpath: str, set_attrs: bool = ..., numeric_owner: bool = ...) -> None: ...  # undocumented
```

(Probably needs a `black` run for proper formatting.)

Reply:
Makes sense! Added comment and reformatted with black.